oss-cloud-backend/serverless.yml

```diff
@@ -47,4 +47,11 @@ functions:
       - http:
           path: contributionStatus
           method: post
+
+  contributorVisibleContributions:
```

Comment:
Can we merge this into `getContributions` lambda and have only one parametrized API to get these.

P.S. should be get, not post. [link](https://www.w3schools.com/tags/ref_httpmethods.asp)

Reply:
Will be merged once search is implemented.